src/plugins/discover/public/utils/format_value.ts

```diff
@@ -53,6 +53,13 @@ export function formatFieldValue(
       .convert(value, usedContentType, converterOptions);
   }
 
+  // format aggregate_metric_double field values
+  if (field.isRolledUpField) {
+    return fieldFormats
+      .getDefaultInstance(KBN_FIELD_TYPES.OBJECT)
+      .convert(value, usedContentType, converterOptions);
+  }
+
```

Comment:
Thx for those changes 🙏 . My question here is, that it should already work without this check, since you extended numeral formatter to also take care of object structure?

Reply:
I guess the reason is, that it's not prettified when removing it, right?

Reply:
Yes, are we ok with the following `JSON.stringify` output? 
 
![67716D7A-2AEF-4B25-B52F-3FDF5C143CF6_4_5005_c](https://user-images.githubusercontent.com/39378793/186682497-67621345-0068-433f-a57b-f0e637049c59.jpeg)